pkg/util/envvar/envvars.go

```diff
@@ -8,6 +8,15 @@ import (
 	corev1 "k8s.io/api/core/v1"
 )
 
+const (
+	AgentImageEnv              = "AGENT_IMAGE"
+	MongodbImageEnv            = "MONGODB_IMAGE"
+	WatchNamespaceEnv          = "WATCH_NAMESPACE"
+	VersionUpgradeHookImageEnv = "VERSION_UPGRADE_HOOK_IMAGE"
+	ReadinessProbeImageEnv     = "READINESS_PROBE_IMAGE"
+	ManagedSecurityContextEnv  = "MANAGED_SECURITY_CONTEXT"
+)
```

Comment:
I'm not sure that this is the correct package for these values, the current `envvars` package has functions related to common functionality around env vars, these are community operator specific - what do you think?

Reply:
Good point, I was actually conflicted on where to put them. Where do you think they will fit best?